floodviz/tests/test_hydrograph_utils.py

```diff
@@ -109,5 +109,11 @@ def test_data_not_list(self):
         self.assertEqual(parse_hydrodata({}), [])
 
     def test_valid_data(self):
-        ret = parse_hydrodata(self.mock_data)
-        self.assertEqual(ret, self.mock_parsed_data)
+            # Delete 'time_mili' key before comparing due to python time conversion inconsistencies.
```

Comment:
You aren't actually deleting the time_mili key so this comment should be removed. I think it would be good to add a test to make sure that parse_hydrodata returns the correct number of list elements too.